lib/itk/queue/consumer.ex

```diff
@@ -271,6 +271,10 @@ defmodule ITKQueue.Consumer do
     if should_retry?(headers) do
       retry(message, channel, meta, subscription, reason)
     else
+      error_handler().handle(queue_name, routing_key, Jason.encode!(message), %RuntimeError{
+        message: reason
+      })
+
```

Comment:
This block doesn’t have the `queue_name` nor `routing_key` destructured also I think it’s better to move it to the `reject/5` function as the `retry_or_die` is defined multiple times and we would need to reuse this block in all the definitions.